### ISSUE TITLE
Restore deleted files from OneDrive

### ISSUE THREAD

**Issue by hingz92** [ChrisTitusTech/win10script#224]
Hello!

I used this great win10script to get my computer freshened up. Unfortunately I uninstalled OneDrive from the computer and apparently all my important work related files where saved on there and not on the normal maps on the computer. Is there anyway on earth I can restore these important deleted files? 

With my humble request and kind regards
/H92

**Comment by jmtgalvez** [comment]
Have you tried reinstalling OneDrive or accessing it through the browser?

**Comment by hingz92** [comment]
Yes, it seems like the OneDrive I had on my computer was a "local account" and not connected to any e-mail-adress which results in no files appearing when I re-installed OneDrive...

**Comment by GnomeBeans** [comment]
One drive has a recycle bin some where maybe look into that.

**Comment by Carterpersall** [comment]
Removing OneDrive when you have files within it will remove the files. Since yours was local (which I didn't even know was possible with OneDrive) your files, unfortunately, likely cannot be recovered after this long. I probably should create a known issues section in the readme so this doesn't end up happening.
